pxfish/pyfish.py

```diff
@@ -111,22 +113,22 @@ def main():
     # have category, look for library or operation type
     category_path = create_named_path(path, args.category)
 
+    user_id = aq.User.where({"login": aq.login}) 
```

Comment:
this call returns an ID?

Reply:
Yes -- I moved it so it only gets retrieved in the push function.